### DISCUSSION TITLE
sdk-python support rabbitmq

### DISCUSSION THREAD

**Discussion by cx-lior-solomon** [cloudevents/sdk-python#200] [Q&A] [answered]
Is the sdk-python support publish a message to rabbitmq?
The sdk-go is indeed support it.
If no, is it a work in progress, when it will be delivered?

**Comment by xSAVIKx** [comment, marked as answer]
Hey, thx for raising this.

AFAIK there is no one actively working on this. You're very welcome to open a PR.

**Reply by cx-lior-solomon** [reply]
Hey @xSAVIKx , do you know if the sdk-python is support it?

I tried, and i'm getting: requests.exceptions.InvalidSchema: No connection adapters were found for 'amqp:
but maybe i'm doing it wrong..

**Reply by xSAVIKx** [reply]
There's no rabbitmq-specific integration, so I guess the answer is `no`. Also, I'm not sure if there's anything specific.

**Reply by cx-lior-solomon** [reply]
rabbitmq using amqp binding,
So the question is if the sdk-python support amqp binding?

**Reply by xSAVIKx** [reply]
[AMQP format](https://github.com/alanconway/cloudevents-spec/blob/master/amqp-format.md) is not supported by the SDK at this moment.

**Reply by cx-lior-solomon** [reply]
got u, thanks!
